### DISCUSSION TITLE
French QR code from "AntiCovid" can't be read

### DISCUSSION THREAD

**Discussion by catshout** [Digitaler-Impfnachweis/certification-apis#93] [Q&A]
Hi all,

I've tried to scan the QR code from France (we live here but I'm working for and travelling to Germany) into the CovPass app. I has been rejected as not valid. The app is being advertized as European wide.

So, how can I scan and process a foreign European country issued QR code (in my case a french one)?

Best
Gerald

**Comment by MikeMcC399** [comment]
@catshout 

There was a similar issue raised in https://github.com/corona-warn-app/cwa-wishlist/issues/560. It seems that compatible codes are planned to be issued in France starting today with the goal of using them starting July 1, 2021.

https://www.ameli.fr/assure/covid-19/vaccination-contre-la-covid-19/lattestation-de-vaccination-certifiee-en-pratique#ancre4

**Quand l’attestation de vaccination sera-t-elle disponible pour voyager ?**

Dès le 23 juin, il sera possible de télécharger son attestation de vaccination certifiée conforme aux normes européennes, sous-titrée en anglais, depuis le téléservice de l’Assurance Maladie. Cette attestation fera partie des preuves autorisées pour voyager au sein de l’Union européenne à compter du 1er juillet prochain.
